Example/FormEntryView.swift

```diff
@@ -45,6 +45,7 @@ final class FormEntryView<V: Validator>: UIView, ValidatorControlDelegate, UITex
         textLabel.textAlignment = .center
         stackView.addArrangedSubview(textLabel)
         
+        textField.autocorrectionType = .no
```

Comment:
Just curious but is this change related? My guess it to stop iOS from suggesting locale based suggestions for this internationalisation support or is there another reason?